product-service/getProductsById.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+import products from './products.json';
+import { headers } from './constants';
+
+export const getById = async event => {
+  try {
+    const { pathParameters } = event;
+    const { id: requiredProductId } = pathParameters;
+    const product = products.find(({ id }) => id === (+requiredProductId));
+
+    if (product) {
+      return {
+        headers,
+        statusCode: 200,
```

Comment:
You may consider using some npm package with predefined status codes constants to visualize which code is used.

Reply:
Thanks for the suggestion.